GW2EIEvtcParser/EncounterLogic/Fractals/SilentSurf/Kanaxai.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace GW2EIEvtcParser.EncounterLogic
 {
-    internal class Kanaxai : FractalLogic
+    internal class Kanaxai : SilentSurf
```

Comment:
You can remove the subcategory init.
You can also remove EncounterIDs.FractalMasks.SilentSurfMask from EncounterID |=